From 8ca07bf06f4a63fe73e7b88bd65e98bf41a028bf Mon Sep 17 00:00:00 2001
From: Jeff Moyer <jmoyer@redhat.com>
Date: Fri, 10 Oct 2014 21:03:14 -0400
Subject: [block] blk-mq: correct a few wrong/bad comments

Message-id: <1412975015-5370-14-git-send-email-jmoyer@redhat.com>
Patchwork-id: 97468
O-Subject: [RHEL7 PATCH 13/34] blk-mq: correct a few wrong/bad comments
Bugzilla: 1146660
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Mike Snitzer <snitzer@redhat.com>

This is a backport of the following commit.  This resolves bug
1146660.

  commit a68aafa5b297d99c2d0c38689089a752126e9e79
  Author: Jens Axboe <axboe@fb.com>
  Date:   Fri Aug 15 13:19:15 2014 -0600

    blk-mq: correct a few wrong/bad comments

    Just grammar or spelling errors, nothing major.

    Signed-off-by: Jens Axboe <axboe@fb.com>

Signed-off-by: Jeff Moyer <jmoyer@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-mq.c b/block/blk-mq.c
index 67a877f..4e8fe2a 100644
--- a/block/blk-mq.c
+++ b/block/blk-mq.c
@@ -1589,7 +1589,7 @@ static int blk_mq_init_hw_queues(struct request_queue *q,
   hctx->tags = set->tags[i];
 
   /*
-   * Allocate space for all possible cpus to avoid allocation in
+   * Allocate space for all possible cpus to avoid allocation at
    * runtime
    */
   hctx->ctxs = kmalloc_node(nr_cpu_ids * sizeof(void *),
@@ -1677,8 +1677,8 @@ static void blk_mq_map_swqueue(struct request_queue *q)
 
  queue_for_each_hw_ctx(q, hctx, i) {
   /*
-   * If not software queues are mapped to this hardware queue,
-   * disable it and free the request entries
+   * If no software queues are mapped to this hardware queue,
+   * disable it and free the request entries.
    */
   if (!hctx->nr_ctx) {
    struct blk_mq_tag_set *set = q->tag_set;
-- 
1.7.1